### ISSUE TITLE
Rendering partial is not working, must add html_safe

### ISSUE THREAD

**Issue by cameronbourgeois** [zurb/inky-rb#27]
In my `mailer.html.inky` I am trying to render a partial `header.html.inky`.

The following code merely renders the escaped html:
```
<container>
  <%= render(partial: "my_mailer/header") %>
</container>
```

By adding `html_safe` to my render method it works fine:
```
<container>
  <%= render(partial: "my_mailer/header").html_safe %>
</container>
```

Is this expected behaviour?

**Comment by killerham** [comment]
I too am having this issue.

**Comment by bobmaerten** [comment]
This has been fixed in the develop branch along many other annoying bugs waiting in PRs, but it seems the project is not moving for a few weeks, so I have to run on a forked repo for now.

**Comment by marcandre** [comment]
Indeed, fixed in master, please release...

**Comment by marcandre** [comment]
Merged, expect a release imminently

**Comment by marcandre** [comment]
Just released 1.3.6.2 :smile:
